homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -53,17 +57,17 @@ def __init__(self, instance, ac_key):
         )
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return if filter needs cleaning."""
-        return self._ac["filterCleanStatus"]
+        return cast(bool, self._ac["filterCleanStatus"])
 
 
 class AdvantageAirZoneMotion(AdvantageAirEntity, BinarySensorEntity):
     """Advantage Air Zone Motion."""
 
     _attr_device_class = BinarySensorDeviceClass.MOTION
 
-    def __init__(self, instance, ac_key, zone_key):
+    def __init__(self, instance: advantage_air, ac_key: int, zone_key: str) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3